src/pages/api/definePassengerType.ts

```diff
@@ -301,6 +302,11 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                             }`,
                         );
                     } else {
+                        const trimmedName = passengerTypeName?.trim();
+                        if (trimmedName) {
+                            await insertPassengerType(getAndValidateNoc(req, res), companions, trimmedName, true);
```

Comment:
Do we need to do a check here to see if there is already a passenger type with the same name? And if so, pass an error message back and redirect back to the page.

Reply:
Good shout